src/element/linearElementEditor.ts

```diff
@@ -686,7 +686,9 @@ export class LinearElementEditor {
 
     const point = element.points[index];
     const { x, y } = element;
-    return rotate(x + point[0], y + point[1], cx, cy, element.angle);
+    return point
+      ? rotate(x + point[0], y + point[1], cx, cy, element.angle)
+      : rotate(x, y, cx, cy, element.angle);
```

Comment:
that's the easy fix. Though I'm not sure what it can break down the line sine this function is used all over the place. While we should ensure this function is type-safe, we should fix the root cause — why we're asking for a point that doesn't exist,, and maybe return `null` for such points (would break a lot of upstream code in ways that I don't know how to handle though.

Reply:
I had the same thoughts. This solution is just sticking plaster.